exercise1/Program.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace exercise1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var dataCenter = "us17";
+            var apiKey = "ed37a130dbadee9671f725ebdf03583a-us17";
+            var listId = "8f5f9903d4";
+            var campaignId = "39494cd8ea";
+
+            Console.WriteLine(GetLists(dataCenter, apiKey, ""));
```

Comment:
Program should enter `email content`, `email to` from user and send the email

Reply:
done